include/eve/module/algo/algo/traits.hpp

```diff
@@ -20,6 +20,31 @@
 
 namespace eve::algo
 {
+  //================================================================================================
```

Comment:
You can save on line by having a single
```
//================================================================================================
//! @addtogroup algo_traits
//! @{
//================================================================================================
```

on top and a single 

```
//================================================================================================
//! @}
//================================================================================================
```

at the end.

Reply:
How about you do this docker things so that unnecessary things are not listed?